### ISSUE TITLE
Namespaced icons  

### ISSUE THREAD

**Issue by lluia** [zopaUK/react-components#20]
Currently, this library exports a [set of wrapper components](https://zopauk.github.io/react-components/#/Components/Icons) around **SVG icons**.

When imported along side other components, it becomes confusing to know what's an **icon** and what's not. It also produces strange name collisions 🧐
```jsx
import {
 Alert,
 AlertBox,
 Profile, 
 CheckMark,
 CheckBoxField
} from '@zopaUk/react-components'

function ProfileAlert() {
  return (
    <AlertBox>
      <div>
        <p><Profile> User – <CheckMark /> verified</p>
      <div>
      <CheckBoxField /> You need to check this
    </AlertBox>
  )
}
```
It seems to me that naming could be cleaner using [JSX dot notation](https://reactjs.org/docs/jsx-in-depth.html#using-dot-notation-for-jsx-type) as a namespace, it would simplify the `import` too:
```jsx
import { Icon, Alert, CheckBox  } from '@zopaUk/react-components'

function ProfileAlert() {
  return (
    <Alert>
      <div>
        <p><Icon.Profile /> User – <Icon.CheckMark /> verified</p>
      <div>
      <CheckBox /> You need to check this
    </Alert>
  );
}
```

**Comment by thegrinder** [comment]
I agree with @lluia, it does make it clearer and easier to import

**Comment by poteirard** [comment]
What if we export the icons with ``ProfileIcon` instead then? I'm not a big fan of namespacing to be honest because you have to remember the structure/tree.

**Comment by lluia** [comment]
@poteirard good point 👌🏻

We could this as a first step? 

Summarising **pros** and **cons** I can see from _namepsacing_:

### Cons
- Need to remember structure to use them
  - _I guess TS integrations will hint to autocomplete with should make this less painful ?_
- `<Icons />` by itself would render `null` I guess 😏

### Pros
- fewer imports ( _as per the example above_ )
- better readability ? ( `<Icons.ZopaLogo />` vs `<ZopaLogoIcon />` )

**Comment by poteirard** [comment]
> fewer imports ( as per the example above )
That's not 100% true you can always do `import * as RC from 'react-components';` and then `RC.ZopaLogo`

**Comment by lluia** [comment]
Addressed on [`2.0`](https://github.com/zopaUK/react-components/pull/54) ☝️
